packages/bdsim/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Bdsim(CMakePackage):
+    """Beam Delivery Simulation (BDSIM) is a C++ program that utilises the Geant4 toolkit
+       to simulate both the transport of particles in an accelerator and their interaction
+       with the accelerator material"""
```

Comment:
```suggestion
       to simulate both the transport of particles in an accelerator and their 
       interaction with the accelerator material"""
```
In order to satisfy `spack style`